apps/explorer/src/components/module/PkgModulesWrapper.tsx

```diff
@@ -89,6 +90,54 @@ function PkgModuleViewWrapper({ id, modules }: Props) {
         });
     };
 
+    const bytecodeContent = [
+        <div
+            key="bytecode"
+            className="grow overflow-auto border-gray-45 pt-5 md:border-l md:pl-7"
```

Comment:
I think we should bake the border into the resize handler instead. This has the odd side-effect of not being able to actually click the border exactly, you need to click to the left of the border.

Reply:
makes sense, I extended so it can accept a `className` now to change the styling directly.

Reply:
Looks like there's still a left border here?

Reply:
Its gone